### PR TITLE
Improves OOC/LOOC/deadchat muted messages

### PR DESCRIPTION
**Pull request by LittleBigKid2000**

### CODE REVIEW

File: code/modules/mob/say.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 /mob/proc/say_dead(var/message)
 	if(!(client && client.holder))
 		if(!config.dsay_allowed)
-			to_chat(src, "<span class='danger'>Deadchat is globally muted.</span>")
+			to_chat(src, "<span class='danger'>HALP ADMINS ABSORBED DEADCHAT</span>")
```
Comment (Crazylemon64) [inline code/modules/mob/say.dm]: N-n-n-nice meme :ok_hand: :+1:

**Comment by taukausanake** [comment]
This just in: All admins are changelings. It's time to break out the incinerators

**Comment by Fox-McCloud** [comment]
These meme got stale over a year ago.

**Comment by LittleBigKid2000** [comment]
Halp Fox absorbing my meme PR

**Comment by Plotron** [comment]
The memes of you and me.

**Pull request closed without merging**